### PR TITLE
reference: update command and default value in pd-control

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? 

This PR updated PD ctl doc to follow the actual usage in code. 

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1818

### Which version does your change affect? 

TiDB 3.0/Dev

### CODE REVIEW

File: dev/reference/tools/pd-control.md [hunk 1]
```diff
@@ -45,13 +49,13 @@ Use TLS to encrypt:
 ### \-\-pd,-u
 
 + PD address
-+ Default address: `http://127.0.0.1:2379`
++ Default address: <http://127.0.0.1:2379>
```
Comment (lilin90) [inline dev/reference/tools/pd-control.md]: @CaitinChen I think you're working on this kind of URLs that may lead to 404. PTAL

Comment (CaitinChen) [reply dev/reference/tools/pd-control.md]: LGTM

File: dev/reference/tools/pd-control.md [hunk 2]
```diff
@@ -21,11 +21,15 @@ As a command line tool of PD, PD Control obtains the state information of the cl
...
-    ./pd-ctl store -d -u http://127.0.0.1:2379
+```bash
+./pd-ctl store -u http://127.0.0.1:2379
+```
 
 Interactive mode:
 
-    ./pd-ctl -u http://127.0.0.1:2379
+```bash
+./pd-ctl -u http://127.0.0.1:2379
```
Comment (lilin90) [inline dev/reference/tools/pd-control.md]:
```suggestion
./pd-ctl -i -u http://127.0.0.1:2379
```

File: dev/reference/tools/pd-control.md [hunk 3]
```diff
@@ -222,11 +226,11 @@ The configuration above is global. You can also tune the configuration by config
...
-
-    ```bash
-    >> config set namespace ts1 leader-schedule-limit 4 // 4 tasks of leader scheduling at the same time at most for the namespace named ts1
-    >> config set namespace ts2 region-schedule-limit 2 // 2 tasks of region scheduling at the same time at most for the namespace named ts2
-    ```
+>
+> ```bash
+> >> config set namespace ts1 leader-schedule-limit 4 // 4 tasks of leader scheduling at the same time at most for the namespace named ts1
+> >> config set namespace ts2 region-schedule-limit 2 // 2 tasks of region scheduling at the same time at most for the namespace named ts2
+> ```
```
Comment (lilin90) [inline dev/reference/tools/pd-control.md]: Thanks for updating the Note format. However, code blocks within a Note often fail to display well. If you want to check whether this one can work, make sure check it on the PingCAP website after this PR is merged.

File: v2.1/reference/tools/pd-control.md [hunk 4]
```diff
@@ -21,11 +21,15 @@ As a command line tool of PD, PD Control obtains the state information of the cl
...
-    ./pd-ctl store -d -u http://127.0.0.1:2379
+```bash
+./pd-ctl store -u http://127.0.0.1:2379
+```
 
 Interactive mode:
 
-    ./pd-ctl -u http://127.0.0.1:2379
+```bash
+./pd-ctl -u http://127.0.0.1:2379
```
Comment (lilin90) [inline v2.1/reference/tools/pd-control.md]:
```suggestion
./pd-ctl -i -u http://127.0.0.1:2379
```

File: v3.0/reference/tools/pd-control.md [hunk 5]
```diff
@@ -20,13 +20,11 @@ As a command line tool of PD, PD Control obtains the state information of the cl
...
 
-Single-command mode:
-
-    ./pd-ctl store -d -u http://127.0.0.1:2379
-
 Interactive mode:
 
-    ./pd-ctl -u http://127.0.0.1:2379
+```bash
+./pd-ctl -u http://127.0.0.1:2379
```
Comment (lilin90) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
./pd-ctl -i -u http://127.0.0.1:2379
```

File: v3.0/reference/tools/pd-control.md [hunk 6]
```diff
@@ -20,13 +20,11 @@ As a command line tool of PD, PD Control obtains the state information of the cl
 
 ## Usage
 
-Single-command mode:
```
Comment (lilin90) [inline v3.0/reference/tools/pd-control.md]: Why remove?

File: v2.1/reference/tools/pd-control.md [hunk 7]
```diff
@@ -21,11 +21,15 @@ As a command line tool of PD, PD Control obtains the state information of the cl
 
 Single-command mode:
 
-    ./pd-ctl store -d -u http://127.0.0.1:2379
+```bash
```
Comment (lilin90) [inline v2.1/reference/tools/pd-control.md]: I'm not sure whether this affects v2.1. Please check.

**Comment by lhy1024** [comment]
LGTM

**Pull request merged**